### ISSUE TITLE
import of xray_vision not found in speckle-plotting

### ISSUE THREAD

**Issue by sbanerjee23** [scikit-xray/scikit-xray-examples#37]
Tried installing from here: https://github.com/Nikea/xray-vision (not available under conda package name in meta file). Tried installing dev branch of scikit-xray. Still no go.

**Comment by ericdill** [comment]
Thanks for the bug report!  Sorting it out now. I'll update when I have a fix.

**Comment by ericdill** [comment]
I just (slightly) updated the speckle-plotting notebook to remove some bugs that you would have hit downstream of your reported bug.  Update to scikit-xray-examples master and try again if you wouldn't mind.

Also, what OS are you on? Linux/mac/windows?

**Comment by sbanerjee23** [comment]
Still the ImportError on mpl_plotting.speckle. Before the first bug report I had installed clint through conda, and changed the viridis cmap to something else. This let me run all cells until "Let's look at what these ring ROIs look like" where mpl_plot was called below. With your commit, including the clint note, I can run everything fine until mpl_plot. 

If xray-vision is a dependency for scikit-xray/examples, it's possible I've installed that incorrectly into my conda environment. I used system python and pip before this, and am now figuring out how to use conda as a package manager and virtualenv replacement. It does show up on my conda list after installing from the Nikea repo. I'm on Linux (Ubuntu 14.04 LTS). 

Last minor note, in the scikit-xray installation doc (https://github.com/scikit-xray/scikit-xray/blob/master/doc/installation.rst) it says Py2 and 3 are supported, but in an anaconda python 3 env, the installation fails after a conda install of scikit-xray. It worked fine in a conda env with python2.7. 

Thanks for the help Eric, much appreciated.

**Comment by ericdill** [comment]
First off, thank you for your patience! I'm fairly certain that this is an issue with how xray-vision was installed (as we have not **really** nicely packaged it up yet).  Can you copy the full stack trace that you see when you experience this error?

```
If xray-vision is a dependency for scikit-xray/examples, it's possible 
I've installed that incorrectly into my conda environment. 
```
It is. I now realize that I have not documented the location of these binaries. I am uploading v0.0.3 of xray-vision to the scikit-xray channel on anaconda.org.  Sorry!  How did you install xray-vision?

If you want to install the xray-vision binary, remove all traces of it from your environment.  `conda remove xray-vision --yes` should work for conda installations. `pip uninstall xray-vision` until it does not find anything will remove things installed with `pip install` or `python setup.py {install, develop}`. Then `conda install xray-vision` **should** install v0.0.3.

```
Last minor note, in the scikit-xray installation doc 
(https://github.com/scikit-xray/scikit-xray/blob/master/doc/installation.rst) 
it says Py2 and 3 are supported, but in an anaconda python 3 env, 
the installation fails after a conda install of scikit-xray. It worked fine
in a conda env with python2.7.
```

Can you copy the full contents of the console output that occurred after you executed `conda install scikit-xray` that produced the failed installation?

**Comment by ericdill** [comment]
Oh. Well. There is a bug in the setup.py file for xray-vision.  That is the origin of all of this.  Give me about 30 minutes and there will be a new tagged version of xray-vision (v0.0.4) that fixes this.  Thanks!

**Comment by ericdill** [comment]
Try now. `conda install -c scikit-xray xray-vision`.

**Comment by sbanerjee23** [comment]
Eric, after reinstalling xray-vision (v0.0.4) the speckle-plotting demo runs without error, thanks for the fix! I had initially installed it from source with `python setup.py install`, but installed from conda this time into a py2.7 env.  

As for the separate py3.x issue, below is the terminal output when I try and install scikit-xray from Anaconda. I only tested Anaconda python 3.3 and 3.5, both giving this 'conflict' message:

`(cedric3)soham@soham-desktop:~$ conda config --add channels scikit-xray`
`Skipping channels: scikit-xray, item already exists`
`(cedric3)soham@soham-desktop:~$ conda install scikit-xray` 
`Fetching package metadata: ......`
`Solving package specifications: ..`
`Error: Unsatisfiable package specifications.`
`Generating hint: `
`[      COMPLETE      ]|###################################################| 100%`


`Hint: the following packages conflict with each other:`
`  - scikit-xray`
`  - python 3.5*`

`Use 'conda info scikit-xray' etc. to see the dependencies for each package.`
`(cedric3)soham@soham-desktop:~$ python --version`
`Python 3.5.0 :: Continuum Analytics, Inc.`

**Comment by ericdill** [comment]
Great!  Thanks for pointing out our error in xray-vison :beers: 

Regarding the error message you received with installing scikit-xray on anaconda 3.3 and 3.5, we were only supporting python 3.4.  Now that python 3.5 is officially released we will start testing 3.5 and get binaries for python 3.5 ready. It is unlikely that we will support python 3.3 unless someone specifically needs it.

Thanks for the patience and feedback!
